src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -53,7 +55,8 @@
  *
  */
 public abstract class CommandLineProgram implements CommandLinePluginProvider {
-    protected final Logger logger = LogManager.getLogger(this.getClass());
+
+    protected final Logger logger = LogManager.getLogger(CommandLineProgram.class);
```

Comment:
I will prefer to maintain the original, because it sets the logger name to the tool name.